JSDemos/Demos/SelectBox/Overview/Angular/app/app.component.ts

```diff
@@ -36,6 +37,10 @@ export class AppComponent {
       key: 'ID',
     });
   }
+
+  onValueChanged(e: any) {
+    notify(`The "${e.value}" item was clicked`);
```

Comment:
The cause of the value change can differ from click. For example it can be changed after "enter" press or event after program option change. I think it's better write smth like `The value is changed to: ${e.value}`.